schema/resourcedefinition.proto

```diff
@@ -73,4 +74,9 @@ message Property {
 
   // If type is OBJECT, this is the name of the object in `messages`.
   string object_type = 5;
+
+  oneof array_type {
+    string array_object_type = 6;
```

Comment:
is this value still needed?

Reply:
This particular field is necessary to define arrays of objects. It'll look way cleaner with the new definition we have above.